packages/editor/src/browser/editor-quick-open.ts

```diff
@@ -0,0 +1,151 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import {
+    QuickOpenModel, QuickOpenItem, QuickOpenMode, PrefixQuickOpenService,
+    OpenerService, QuickOpenGroupItemOptions, QuickOpenItemOptions,
+    QuickOpenHandler, QuickOpenOptions
+} from '@theia/core/lib/browser';
+import { MessageService } from '@theia/core/lib/common/message-service';
+import URI from '@theia/core/lib/common/uri';
+import { LabelProvider } from '@theia/core/lib/browser/label-provider';
+import { Command } from '@theia/core/lib/common';
+import { CancellationTokenSource } from '@theia/core/lib/common';
+import { EditorManager } from './editor-manager';
+import { EditorWidget } from './editor-widget';
+
+export const SHOW_ALL_OPENED_EDITORS: Command = {
+    id: 'workbench.action.showAllEditors',
+    category: 'View',
+    label: 'Show All Opened Editors'
+};
+
+@injectable()
+export class EditorQuickOpenService implements QuickOpenModel, QuickOpenHandler {
```

Comment:
please always name a file after a class or other way around, it's a part of our coding guidlines: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names

Reply:
Thanks! Will fix it.